helpers/base.py

```diff
@@ -170,3 +170,25 @@ def ghe_json_message(rec):
         return
 
     return message_rec
+
+def select_key(data, search_key, results):
+    """ Recursively search for a given key and return all values
+    Args:
+        data (dict, list)
+        search_key (string)
+        results (array)
+
+    Returns:
+        (list) all values
+    """
+    if not hasattr(data, 'items'):
```

Comment:
What's the goal of this check? To see if `data` is a dict? Or to check if it's iterable?

Reply:
If it's the former, please use `if not isinstance(data, dict): ...`

Reply:
its the latter (it could be a dict or a list)

Reply:
Okay in any case, lists don't have the `items` attribute, so you can pass a tuple to isinstance list: `isinstance(data, (dict, list))`

Reply:
Done, thanks for the suggestion :)